datasets/_global/gem_energy_ownership/crawler.py

```diff
@@ -14,10 +18,7 @@
     "gem_parents",
     "gem_parents_ids",
 ]
-
-# Unique entity types
-# {"person", "unknown entity", "state", "legal entity", "arrangement", "state body"}
-
+ALIAS_SPLITS = ["[former],", "[former]", "[former name]", "(former)"]
```

Comment:
since all of these are former, it might be nice to put them into `previousName` instead.

Reply:
my bad - I didn't realise this was originally connected to alias and the field name was name_other which suggests that most of the time, it'll actually be aliases but sometimes there are former name splits.

This kinda highlights an issue with declaring and extracting lots of values at the top, and using them later. I've changed the ID extracts to be mostly in line with where we use them. having the original key and our prop in close proximity makes it easier to see how we're mapping it without scrolling up and down a page worth of code.